config/testgrids/kubernetes/sig-windows/config.yaml

```diff
@@ -0,0 +1,19 @@
+dashboard_groups:
+- name: sig-windows
+  dashboard_names:
+    - sig-windows-networking
```

Comment:
Looks like `sig-windows-networking` dashboard is never used:

```
    config_test.go:196: Dashboard sig-windows-networking: - Must have more than one dashboardtab
```

Reply:
The idea was to move the tabs that were located at https://github.com/kubernetes/test-infra/pull/18123/files#diff-7e7de6e8742581c31dca06d3ce098138L519  to networking tab.   

The tests don't seem to have a job configuration associated with them. I can't find the config anywhere in this repository.  The tests on the current testgrid also look like they are not active: https://testgrid.k8s.io/sig-windows#flannel-l2bridge-windows-master.  I can push an update that removes this but would like to know where those test jobs are configured so we don't loose them if they are still around.

I believe @adelina-t and team were making some updates to those tests?

Reply:
update after talking with @adelina-t offline.  this configurations are just for the reporting.  The jobs are run elsewhere and artifacts are published to the storage.  Going to keep these configurations for now until the configuration for the jobs move to the windows-testing repository.

The ovn/ovs tests configurations can be removed.

Reply:
Could you try this config:

```diff
diff --git a/config/testgrids/kubernetes/sig-windows/config.yaml b/config/testgrids/kubernetes/sig-windows/config.yaml
index c7e53215e9..9be642cea5 100644
--- a/config/testgrids/kubernetes/sig-windows/config.yaml
+++ b/config/testgrids/kubernetes/sig-windows/config.yaml
@@ -7,6 +7,7 @@ dashboard_groups:
     - sig-windows-presubmit
     - sig-windows-gce
     - sig-windows-azure
+    - sig-windows-networking
 
 dashboards:
 - name: sig-windows-releases
@@ -15,3 +16,17 @@ dashboards:
 - name: sig-windows-presubmit
 - name: sig-windows-gce
 - name: sig-windows-azure
+- name: sig-windows-networking
+  dashboard_tab:
+  - name: flannel-l2bridge-windows-master
+    description: Runs Windows e2e tests on K8s clusters on Openstack vms with Flannel CNI in l2bridge network mode.
+    test_group_name: ci-kubernetes-e2e-flannel-l2bridge-master-windows
+  - name: flannel-overlay-windows-master
+    description: Runs Windows e2e tests on K8s clusters on Openstack vms with Flannel CNI in overlay network mode.
+    test_group_name: ci-kubernetes-e2e-flannel-overlay-master-windows
+  - name: containerd-l2bridge-windows-master
+    description: Runs Windows e2e tests on K8s clusters with Containerd and Flannel CNI in l2bridge network mode.
+    test_group_name: ci-kubernetes-e2e-sdnbridge-ctrd-master-windows
+  - name: containerd-overlay-windows-master
+    description: Runs Windows e2e tests on K8s clusters with Containerd and Flannel CNI in overlay network mode.
+    test_group_name: ci-kubernetes-e2e-sdnoverlay-ctrd-master-windows
```

Reply:
Thanks.  Was able to get it to pass using https://github.com/kubernetes/test-infra/blob/5994b843c71c0ea1da1e50682e9db58c77390dc3/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml#L57 as an example